Simplify README and link to wiki 📖 (#76)
sunny/actor: README.md

```rb
actor = BuildGreeting.call
actor.greeting # => "Have a wonderful day!"
actor.greeting? # => true
```

### Defaults

> Inputs can become optional if you provide a default:

Author: ```suggestion
Inputs can be optional by providing a default:
```

You will find that cutting your business logic into single purpose actors will
make it easier for you to test your application.

## FAQ

Howtos and frequently asked questions can be found on the
[wiki](https://github.com/sunny/actor/wiki).

## Thanks

> This gem is influenced & compatible with

Author: ```suggestion
This gem is influenced by (and compatible with)
```
